tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm

```diff
@@ -0,0 +1,79 @@
+################################################################################
+FROM quay.io/pypa/manylinux2014_x86_64 as builder
+################################################################################
+
+# Install devtoolset build dependencies
+COPY setup.packages.rocm.sh setup.packages.rocm.sh
+COPY builder.packages.rocm.txt builder.packages.rocm.txt
+RUN /setup.packages.rocm.sh /builder.packages.rocm.txt
+
+# Install devtoolset-7 in /dt7 with glibc 2.12 and libstdc++ 4.4, for building
+# manylinux2010-compatible packages. Scripts expect to be in the root directory.
```

Comment:
Let's remove all of these commented out lines

Reply:
done